### DISCUSSION TITLE
The Legacy Edition

### DISCUSSION THREAD

**Discussion by rcarmo** [piku/piku#285] [Announcements]
This is a snapshot of the source tree that should be suitable for Python 3.5-3.7 on Debian 9/10 and Ubuntu 16.04/18.04.

<hr /><em>This discussion was created from the release <a href='https://github.com/piku/piku/releases/tag/legacy'>The Legacy Edition</a>.</em>

**Comment by chr15m** [comment]
Thanks for creating The legacy Edition, Rui. Just a minor thing but should we not give it a versioned name (such as including the year/month) as we will likely have to do this again when future Python versions are inevitably deprecated?

**Comment by rcarmo** [comment]
We can re-tag it as 0.9 or something once the 3.7/3.8 cleanup is done and release that as 1.0, or use 3.5 and align with the Python version.

I am not particularly keen on any versioning schema (although dates seem to be more logical in something that is going to align to both OS and Python releases), so will take anything that has some form of consensus.
